app/src/stores/useToastStore.ts

```diff
@@ -0,0 +1,29 @@
+import { defineStore } from 'pinia'
+import { ref } from 'vue'
+import { ToastType } from '@/types'
+
+interface Toast {
+  message: string
+  type: ToastType
+  timeout: number
+}
+
+export const useToastStore = defineStore('_toast', () => {
+  const toasts = ref<Toast[]>([])
+
+  const addToast = (
+    message: string,
+    type: ToastType = ToastType.Success,
+    timeout: number = 5000
+  ) => {
+    toasts.value.push({ message, type, timeout })
+    setTimeout(() => {
+      toasts.value.shift()
```

Comment:
This algorithm is not good. 

When you have different toast with different timeout you can remove one before his timeout finish